docs/data/date-pickers/custom-components/custom-components.md

```diff
@@ -219,6 +219,34 @@ Each component comes with its own toolbar (`DatePickerToolbar`, `TimePickerToolb
 
 {{"demo": "ToolbarComponent.js"}}
 
+## Calendar header
+
+The calendar header is available on any component that renders a calendar to select a date or a range of dates.
+It allows the user to navigate through months and to switch to the month and year views when available.
+
+### Component props
+
+You can pass props to the calendar header as shown below:
+
+{{"demo": "CalendarHeaderComponentProps.js", "defaultCodeOpen": false}}
```

Comment:
I have no idea what we could demonstrate here :grimacing: 
The component have no `ExportXXX` props, except for the one of the `PickersArrowSwitcher` which is already documented